grpc/src/main/java/io/stargate/grpc/service/ValuesHelper.java

```diff
@@ -43,32 +44,43 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Objects;
+import javax.annotation.Nonnull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class ValuesHelper {
+  private static final Logger LOG = LoggerFactory.getLogger(ValuesHelper.class);
+
   public static BoundStatement bindValues(Prepared prepared, Values values, ByteBuffer unsetValue)
       throws StatusException {
     final List<Column> columns = prepared.metadata.columns;
     final int columnCount = columns.size();
     final int valuesCount = values.getValuesCount();
     if (columnCount != valuesCount) {
-      throw Status.FAILED_PRECONDITION
-          .withDescription(
-              String.format(
-                  "Invalid number of bind values. Expected %d, but received %d",
-                  columnCount, valuesCount))
-          .asException();
+      StatusException statusException =
+          Status.FAILED_PRECONDITION
+              .withDescription(
+                  String.format(
+                      "Invalid number of bind values. Expected %d, but received %d",
+                      columnCount, valuesCount))
+              .asException();
+      LOG.error(getErrorMessage(Status.FAILED_PRECONDITION, statusException), statusException);
+      throw statusException;
     }
     final List<ByteBuffer> boundValues = new ArrayList<>(columnCount);
     List<String> boundValueNames = null;
     if (values.getValueNamesCount() != 0) {
       final int namesCount = values.getValueNamesCount();
       if (namesCount != columnCount) {
-        throw Status.FAILED_PRECONDITION
-            .withDescription(
-                String.format(
-                    "Invalid number of bind names. Expected %d, but received %d",
-                    columnCount, namesCount))
-            .asException();
+        StatusException statusException =
+            Status.FAILED_PRECONDITION
+                .withDescription(
+                    String.format(
+                        "Invalid number of bind names. Expected %d, but received %d",
+                        columnCount, namesCount))
+                .asException();
+        LOG.error(getErrorMessage(Status.FAILED_PRECONDITION, statusException), statusException);
```

Comment:
This may make sense for now, to get more information, but aren't these client errors triggered by incorrect usage of gRPC client by customers? Since we alert on ERRORs this can lead to lots of noise.